### PR TITLE
Add a function to create traversals

### PR DESCRIPTION
**Pull request by aspiwack**
I'm assuming, here, that traversals are built from the same type as
`Traverse`. I believe this to be the right type, which I explain in
some details in #220.

Depends on  #219 #221 .

### CODE REVIEW

**Comment by aspiwack** [comment]
There is a bit of noise due to the dependency on two different PRs, so either review just the last commit, or wait until #219 is merged.

File: src/Data/Profunctor/Linear.hs
Comment (Divesh-Otwani) [inline src/Data/Profunctor/Linear.hs:65]: I like this much better actually.

**Review by Divesh-Otwani — APPROVED** [review]

File: src/Data/Profunctor/Linear.hs
Comment (aspiwack) [reply src/Data/Profunctor/Linear.hs:65]: In principle, I like the previous one better. Like, it's theoretically less ad hoc. But on the other hand, it's also less general in a sense.

**Pull request merged**
